examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -104,13 +109,16 @@ class _CustomValueIndicatorShape extends SliderComponentShape {
       end: _slideUpHeight,
     );
     final double size = _indicatorSize * sizeTween.evaluate(enableAnimation);
-    final Offset slideUpOffset = Offset(0.0, -slideUpTween.evaluate(activationAnimation));
+    final Offset slideUpOffset =
```

Comment:
Please restore the formatting here.

... I'm going to stop providing this feedback. Please restore the formatting for all of the code that wasn't actually changed. 

For new/changed code: Flutter doesn't use dartfmt, so you'll have to format things by hand.  If you're careful you can just work by analogy with existing code. The complete style guide is here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo